### PR TITLE
fix(ui) Fix incorrectly centered text in muted box

### PR DESCRIPTION
**Pull request by markstory**
When this was converted to flexbox I missed a scenario and the complete ignore was incorrectly laid out. I've also fixed more double borders that were showing up on muted and resolved issues.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx [hunk 1]
```diff
@@ -191,18 +191,14 @@ class GroupEventDetails extends React.Component<Props, State> {
                 location={location}
               />
             )}
-            {group.status !== 'unresolved' && (
-              <div className="issue-status">
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx]: There weren't any styles attached to this classname.

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx [hunk 2]
```diff
@@ -191,18 +191,14 @@ class GroupEventDetails extends React.Component<Props, State> {
...
-                  <MutedBox statusDetails={group.statusDetails} />
-                )}
-                {group.status === 'resolved' && (
-                  <ResolutionBox
-                    statusDetails={group.statusDetails}
-                    orgId={organization.slug}
-                  />
-                )}
-              </div>
+            {group.status === 'ignored' && (
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx]: I know that this is probably out of scope of this PR, but should we make enum out of these?

**Review by matejminar — APPROVED** [review]
Looks good to me :)

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Review by billyvg — APPROVED** [review]
Would it be worth having a [visual] snapshot of this state?

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx (hunk 2 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx]: An enum for these could be helpful.

**Pull request merged**